### DISCUSSION TITLE
Machine learning running but no indexing on self-hosting

### DISCUSSION THREAD

**Discussion by nothingbutlucas** [ente-io/ente#4064] [Q&A] [answered]
Hi there!

So, I selfhosted ente photos, everything is working but the machine learning. I tried on mobile and desktop. It says that it is running but nothing is processed

![imagen](https://github.com/user-attachments/assets/39249cb3-0911-4710-adbf-f67ed4d5b5cd)

**Comment by mnvr** [comment, marked as answer]
It is likely not able to download the images to index them. You will see the corresponding errors in the logs - the ML processing will just be stuck on the fetching step.

To resolve download issues, this might help - https://help.ente.io/self-hosting/guides/configuring-s3

**Reply by nothingbutlucas** [reply]
Hi! Thanks you for your answer! It seems that I had an error on my `compose.yaml`
For some reason I declared `- .museum.yaml:/museum.yaml:ro` instead of `- ./museum.yaml:/museum.yaml:ro`

**Reply by MNickelson** [reply]
Which logs in particular? I'm seeing the same behavior but have the correct museum.yaml declaration.
In `docker compose logs` I don't see anything that looks like a failure or error.

The desktop client has
```
[2024-12-07T21:16:37-05:00] [main] [info] Attempting to handle request to open URL: --allow-file-access-from-files
[2024-12-07T21:16:37-05:00] [rndr] [info] Ignoring unhandled open request for URL --allow-file-access-from-files
```
but nothing else.

edit: Never mind. `killall ente` and restarting the program got it running.
